### DISCUSSION TITLE
ability to set default values for vj-text

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1914] [Ideas]
It would be very useful to be able to set the default line spacing and bottom margin for Text Blocks class vj-text. Those are two very common requests, and that is a very commonly used block.

**Comment by meetmandeep** [comment]
How about setting the default line spacing or bottom margin on the text styles? A / B / C / etc...

This allows you to use different styling for different needs.

**Comment by eswanzey** [comment]
You are right, predefined styles go a long way towards filling the need. I haven’t used them much but now will. It’s very nice that the text block assumes the name of the applied style (oh, that’s why they’re all labeled “Text – A”).

It seems like given the styles custom subcategory names in settings should probably be a punch list item at the start of any project. Or does Vanjaro automagically update all references if style “A” is changed to “AAA” and so on down the line? That would be a great thing.

**Comment by meetmandeep** [comment]
Style names A, B, C are static and never change. Add new features to an existing style will not overwrite your custom changes when upgrading Vanjaro.

**Comment by eswanzey** [comment]
I see that new instances of various blocks will exhibit revisions to "style A," which is expected behavior. But revising style A doesn’t affect existing blocks that use that style. I wonder if that is considered to be a bug, or is intended.

**Reply by Gaganbird** [reply]
Can you please confirm if existing block is having "paragraph-style-1" class.

**Comment by eswanzey** [comment]
Yes, on the enclosing div.
